src/app/src/components/MeetTheFishButton.js

```diff
@@ -1,55 +1,104 @@
 import React, { useState } from 'react';
-import styled from 'styled-components';
+import styled, { keyframes } from 'styled-components';
 import { func, number, bool } from 'prop-types';
 
 import { CatalogFish } from '../util/CatalogFish';
 import { FISH_MODAL_OPEN_DELAY } from '../util/constants';
 
 import FishNames from './FishNames';
 
-const FishImage = styled.img`
-    max-width: 350px;
-    max-height: 400px;
-    transition: transform 1s linear;
-
-    &.large {
-        transform: scale(1.5);
+const fishBounce = keyframes`
+    0% {
+        transform: translateY(0);
+    }
+  
+    50% {
+        transform: translateY(3%);
+    }
+  
+    100% {
+        transform: translateY(0%);
     }
 `;
 
 const FishButton = styled.button`
     background-color: transparent;
     border: none;
+    position: relative;
+    transition: all 0.35s ease-in;
```

Comment:
The transition can be a bit jumpy at this speed and amount of growth. I liked the UX at a slower transition and a bit more growth, tested from my local branch on the ELO. Try 0.5s speed and an `.is-selected` transform scale of 1.2 or 1.25 and see what you think

Reply:
Alex and I worked on changes together on my computer & the ELO. As per our discussion, I pushed up said changes to this branch.